flipperdevices/flipperzero-firmware
# [FL-2198], [FL-2161] NFC emulation refactoring

# What's new

- Show reader commands in UID emulation mode
- New nfc emulation approach
- Fix bt settings unintended switch to forget devices scene

# Verification 

- Verify flipper displays reader commands in UID emulation
- Verify Mifare Ultralight emulation works as usual

# Checklist (do not modify)
